make/autoconf/jdk-options.m4

```diff
@@ -302,8 +302,24 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_DEBUG_SYMBOLS],
       [AS_HELP_STRING([--with-native-debug-symbols],
       [set the native debug symbol configuration (none, internal, external, zipped) @<:@varying@:>@])],
       [
+        if test "x$OPENJDK_TARGET_OS" = xaix; then
+          if test "x$withval" = xexternal || test "x$withval" = xzipped; then
+            # We support external for AIX now
+          fi
+        fi
```

Comment:
This does nothing at all and can be removed.

Reply:
:-)